tools/README.md

```diff
@@ -19,5 +19,12 @@ Note: there is a Docker image ([boutiques/boutiques](https://hub.docker.com/r/bo
     * Print a command line based on parameters from an input file: `docker run --rm -v $PWD:/work -w /work boutiques/boutiques localExec.py -i input_param_file.csv ./tool.json`
   * Check the help page (`-h` option) for more options and documentation.
 
-3. `pegasus-boutiques`: a python API to generate [Pegasus](https://pegasus.isi.edu) workflows from Boutiques JSON descriptors.
+3. `invocationSchemaHandler.py`: a Python script to generate and test invocation schemas (i.e. JSON schemas for input values) associated to a Boutiques application
+     descriptor and validating input data examples with respect to these invocation schemas.
+  * Usage examples:
+    * Generates an invocation schema: `python invocationSchemaHandler.py -i toolDescriptor.json`
+    * Validate input data (i.e. tool parameter arguments) with implicit invocation schema generation: `python invocationSchemaHandler.py -i toolDescriptor.json -d tool.exampleInputs.json`
```

Comment:
That's also a little confusing to me because the invocation schema in that case will not be displayed or written, it be generated and used to check the specific input values in `tool.exampleInputs.json`.

How about just "validate input data"?